lib/auth/windows/windows.go

```diff
@@ -257,6 +257,10 @@ var (
 	// this value was determined empirically based on how AD CA's Enterprise CA issues these
 	// certificates post the May 10, 2022 Windows update.
 	ADUserMappingInternalOID = append(ADUserMappingExtensionOID, 1)
+
+	//LicenseOID is extension signaling the license type of Teleport build.
+	//It should take values "oss" or "ent" (the values returned by modules.GetModules().BuildType())
+	LicenseOID = asn1.ObjectIdentifier{1, 3, 1, 0, 0, 5}
```

Comment:
It looks like we've kept all custom OIDs in `ca.go` up to this point, and use the `1.3.9999.x.y` namespace:

https://github.com/gravitational/teleport/blob/80050a1db9de0e73700d78aa0c77a8a3a3a8fc8b/lib/tlsca/ca.go#L335-L489

so let's stick with that pattern unless we have a particularly good reason not to.

Reply:
Yep, makes sense, I've changed it